lib/examples/rocket/src/lib.rs

```diff
@@ -0,0 +1,33 @@
+mod error;
+pub mod person;
+
+use rocket::serde::{Deserialize, Serialize};
+use std::env;
+use std::sync::Arc;
+use surrealdb::engine::any;
+use surrealdb::opt::auth::Root;
+use surrealdb::Surreal;
+
+
+#[derive(Serialize, Deserialize)]
+#[serde(crate = "rocket::serde")]
+pub struct Person {
+	name: String,
+}
+
+pub type Db = Arc<Surreal<any::Any>>;
```

Comment:
Why is `Arc` necessary here? `Surreal<C>` is both `Send` and `Sync`. It's also `'static`.